examples/react/algolia/src/styles.css

```
@@ -22,14 +22,23 @@
+  border: 0;
   color: blue;
+  font: inherit;
   padding-left: 20px;
   padding-top: 20px;
   cursor: pointer;
   font-weight: bold;
   text-transform: uppercase;
 }
 
+.search-more:focus-visible {
+  outline: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)